apps/web-giddh/src/app/accounting/accouting-sidebar/accounting-sidebar.component.html

```diff
@@ -25,15 +25,34 @@
             >{{ commonLocaleData?.app_keys?.f6 }}: {{ commonLocaleData?.app_voucher_types?.receipt }}
         </a>
     </li>
-
     <li>
         <a
             class="cp"
             (click)="setSelectedPage(accountingVouchers.JOURNAL, 'voucher', 'purchases')"
             [ngClass]="{ 'active-parent': (selectedVoucher | lowercase) === accountingVouchers.JOURNAL }"
-            >F7: Journal</a
+            >{{ commonLocaleData?.app_keys?.f7 }}: {{ commonLocaleData?.app_voucher_types?.journal }}</a
+        >
+    </li>
+    <li>
+        <a
+            class="cp"
+            (click)="setSelectedPage(accountingVouchers.SALES, 'voucher', 'sales')"
```

Comment:
add href atrribute in all <a></a> anchor tag

Reply:
href="javascript:;"